packages/reporter/src/commands/command.tsx

```diff
@@ -38,7 +38,8 @@ export const formattedMessage = (message: string) => {
   const matchingText = searchText.find((text) => message.includes(text))
   const textToConvert = [split[0].trim(), ...(matchingText ? [matchingText] : [])].join(' ')
   const spaceEscapedText = textToConvert.replace(/^ +/gm, (initialSpaces) => '&#32;'.repeat(initialSpaces.length)) // &#32 is the HTML entity for a space
-  const converted = md.renderInline(spaceEscapedText)
+  const underscorePreservedText = spaceEscapedText.replace(/_+/gm, (initialUnderscores) => '&#95;'.repeat(initialUnderscores.length)) // &#32 HTML entity for underscore
+  const converted = md.renderInline(underscorePreservedText)
```

Comment:
can we add a test for this?

Reply:
I updated one of the existing tests and added a clarifying comment